lib/jellyfish_web/api_spec/component/hls.ex

```diff
@@ -31,9 +31,14 @@ defmodule JellyfishWeb.ApiSpec.Component.HLS do
         persistent: %Schema{
           type: :boolean,
           description: "Whether the video is stored after end of stream"
+        },
+        subscribeMode: %Schema{
+          type: :string,
+          description: "Whether the HLS should subscribe to tracks automatically or manually.",
```

Comment:
```suggestion
          description: "Whether the component subscribes to tracks automatically or manually",
```

Reply:
it's only hls component feature so that would be incorrect.